### PR TITLE
Add git commands to template

### PR DESCRIPTION
**Pull request by arthurnn**
More convenient to init the repo with git.

### CODE REVIEW

**Comment by arthurnn** [comment]
review @dwradcliffe @byroot

**Comment by byroot** [comment]
Is the secrets.yml ignored in dev though?

**Comment by arthurnn** [comment]
> Is the secrets.yml ignored in dev though?

I guess we should use ENV variables even for dev, and probably use foreman locally? WDYT ?

**Comment by byroot** [comment]
I'm not super familiar with foreman in dev. Does it load env from a .env file or something? If so then yes :+1:

**Comment by arthurnn** [comment]
Yes, it loads the `.env` file.. we can create that file on the template and ignore it on git.

**Comment by byroot** [comment]
Go for it man!

**Comment by arthurnn** [comment]
Ok done.. So this allows some good things:
- `foreman start` will now start sidekiq and rails in development, so we can use sidekiq locally
- foreman will use the .env file. Which is also used for the docker container I am creating. So if you already have the `.env` file from a development version, you can just copy/paste it to the container version. Not that useful, but at least is consistent. 
- We can push the secrets file now, as it doesnt contain sensitive data.

File: template.rb [hunk 1]
```diff
@@ -8,12 +8,11 @@
 
 gem 'sidekiq'
 gem 'thin'
-gem 'shipit-engine'
+gem 'shipit-engine', github: 'Shopify/shipit-engine'
```
Comment (gmalette) [inline template.rb]: should probably change before merge?

**Comment by gmalette** [comment]
nice! I can't wait

**Pull request merged**